### PR TITLE
release-notes: Fix typo, whitespace cleanups

### PR DESCRIPTION
**Pull request by perlun**
Another minor fix, noted that some Norwegian :norway: had slipped in to the release notes.

### CODE REVIEW

File: release-notes.md [hunk 1]
```diff
@@ -86,13 +86,13 @@ Byte Buddy release notes
...
 
-### 9. Mars 2021: version 1.10.22
+### 9. March 2021: version 1.10.22
 
 - Refactor `JavaConstant` API and fix handle resolution.
 - Adjust bootstrap method validation to account for possible dynamic constants.
 - Make class loaders parallel capable.
 - Correct code for attaching to Windows VMs with 32 bit.
-- Allow configuration for Gradle plugin to consider class path as incremental. 
+- Allow configuration for Gradle plugin to consider class path as incremental.
```
Comment (perlun) [inline release-notes.md:95]: My editor is set to trim trailing whitespace on file save, so a bunch of these slipped in as well. Can move them to a separate PR if you so prefer.

**Comment by raphw** [comment]
Great, thanks!
